jck/jck-semiauto-updater/jckupdater.sh

```diff
@@ -222,23 +302,24 @@ get_JAVA_SDK(){
 #Unpack downloaded jar files 
 extract() {
 	cd $WORKSPACE/jckmaterial
-	
-  	echo "install downloaded resources"
-
-	for f in $WORKSPACE/jckmaterial/*.jar; do
-		echo "Unpacking $f:"
-		
-		#using default java on machine for local
-		if [[ $JAVA_HOME != "" ]] ; then
-			$JAVA_HOME/bin/java -jar $f -install shell_scripts -o $WORKSPACE/unpackjck
-		else
-			$JAVA_SDK_PATH/bin/java -jar $f -install shell_scripts -o $WORKSPACE/unpackjck
-		fi
+	if [ $result -ne 0 ]; then
```

Comment:
if block should be added in https://github.com/adoptium/aqa-tests/blob/5e143d4d9370c9306fa5830d206cc80b1db58d22/jck/jck-semiauto-updater/jckupdater.sh#L440-L443

Reply:
yes will add if block here